tools/travis/run_lxd_container.sh

```diff
@@ -32,8 +32,7 @@ name="$1"
 lxc="/snap/bin/lxc"
 
 echo "Starting the LXD container."
-# FIXME switch back to unprovileged once LP: #1709536 is fixed
-$lxc launch --ephemeral --config security.privileged=true --config security.nesting=true ubuntu:xenial "$name"
+$lxc launch --ephemeral --config security.nesting=true ubuntu:xenial "$name"
```

Comment:
You say "unconfined containers" but these are no longer privileged-- did you mean "unprivileged"? LP: #1709536 does seem to be fixed so, great!